script/create_admin.rb

```diff
@@ -65,7 +65,7 @@ def multi_gets(all_text = "")
     PASSFILE
   end
 
-  puts "==== #{a.login}.txt"
+  puts "==== #{a.login}_staging.txt"
```

Comment:
I think this is missing some logic to check which environment we're running the script on.